notebooks/book2/05/minimize_kl_divergence.ipynb

```diff
@@ -0,0 +1,284 @@
+{
```

Comment:
<p>For first plot, loss does not looks converged completely, I think you should increase number of ierations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>